crypto/pkcs12/pkcs12-example.c

```diff
@@ -87,42 +86,50 @@ int main(int argc, char** argv)
     ret = wc_PKCS12_parse(pkcs12, "wolfSSL test", &keyDer, &keySz,
             &certDer, &certSz, &list);
     printf("return value of parsing pkcs12 = %d %s\n", ret, (ret == 0)? "SUCCESS": "FAIL");
-    if (ret != 0 || keyDer == NULL || certDer == NULL) {
+    if (ret != 0) {
         printf("\t error parsing pkcs12\n");
         wc_PKCS12_free(pkcs12);
         return -1;
     }
 
     /* print out key and cert found */
-    printf("HEX of Private Key Read (DER format) :\n");
-    for (i = 0; i < keySz; i++) {
-        if (i != 0 && !(i%16)) printf("\n");
-        printf("%02X", keyDer[i]);
-    }
-    printf("\n");
-
-    printf("\nHEX of Certificate Read (DER format) :\n");
-    for (i = 0; i < certSz; i++) {
-        if (i != 0 && !(i%16)) printf("\n");
-        printf("%02X", certDer[i]);
-    }
-    printf("\n");
-
     if (keyDer != NULL) {
+        printf("HEX of Private Key Read (DER format) :\n");
+        for (i = 0; i < keySz; i++) {
+            if (i != 0 && !(i%16)) printf("\n");
```

Comment:
A question: I don't see anything explicitly forbidding use of `if` without brackets in the coding standards (and I know you didn't create this code, just moved it)...  There is an implicit mention of using "K&R" style and `if` as an example...  Do we generally allow the "no-bracket" formatting for `if` statements? .. maybe only when there isn't an `else` statement?